Simplenote/Classes/SPTagsListViewController.m

```diff
@@ -588,28 +622,25 @@ - (BOOL)textFieldShouldReturn:(UITextField *)textField {
 #pragma mark - SidePanelDelegate
 
 - (void)containerViewControllerDidHideSidePanel:(SPSidebarContainerViewController *)container {
-    
     self.bVisible = NO;
     [self setEditing:NO canceled:YES];
     
 }
 - (void)containerViewControllerDidShowSidePanel:(SPSidebarContainerViewController *)container {
-    
     self.bVisible = YES;
 }
 
 - (BOOL)containerViewControllerShouldShowSidePanel:(SPSidebarContainerViewController *)container {
-
     dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
I realize this was here before, but I'm wondering why a view controller method needs to dispatch its work asynchronously to the main thread?  Is this called from a background thread somehow? 😬

Reply:
This one was **heavily** modified in another PR, leaving it for now this way (because this entire API changed 100%!)